### ISSUE TITLE
Tab key support is missing for menu options.

### ISSUE THREAD

**Issue by v-pavanp** [Microsoft/vscode#6986]
- VSCode Version: 1.2.0-alpha
- OS Version: Windows10

I observed tab key is not working as expected for menu options. When user click on File menu and try to switch to other menu options using tab key then nothing is happening.
Expecting it should behave similar to Visual studio:

![tabsequenceissues](https://cloud.githubusercontent.com/assets/17735796/15624493/bc6569e4-243d-11e6-9d13-063713e9288f.gif)

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/17278
